### PR TITLE
(DPS-2899) Send scroll info to GA from docs

### PR DESCRIPTION
**Pull request by arturfracala**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/DPS-2899)
- [Jira](https://livechatinc.atlassian.net/browse/DPS-2899)

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -74,3 +74,26 @@ export const setUrlParams = (section) => {
...
+export const getCategoryTitle = (menuItems) => {
+  const pathname = window.location.pathname;
+
+  var categoryTitle = "";
+
+  const iterate = (obj) => {
+    if (obj.url === pathname) {
+      categoryTitle = obj.title;
+    }
+    if (obj.items && obj.items.length > 0) {
```
Comment (dominikslawkowski) [inline src/utils/index.js:87]: maybe just `obj.items?.length > 0`

File: src/utils/index.js [hunk 2]
```diff
@@ -74,3 +74,26 @@ export const setUrlParams = (section) => {
     window.history.replaceState(null, null, params);
   }
 };
+
+export const getCategoryTitle = (menuItems) => {
+  const pathname = window.location.pathname;
+
+  var categoryTitle = "";
+
+  const iterate = (obj) => {
```
Comment (dominikslawkowski) [inline src/utils/index.js:83]: It could be called `findTitle`?

Comment (dominikslawkowski) [reply src/utils/index.js:83]: and maybe just `item` instead of `obj` to stay consistent?

**Review by dominikslawkowski — APPROVED** [review]

File: src/utils/index.js [hunk 3]
```diff
@@ -74,3 +74,26 @@ export const setUrlParams = (section) => {
     window.history.replaceState(null, null, params);
   }
 };
+
+export const getCategoryTitle = (menuItems) => {
+  const pathname = window.location.pathname;
+
+  var categoryTitle = "";
```
Comment (jakubsikora) [inline src/utils/index.js:81]:
```suggestion
  let categoryTitle = "";
```

**Review by jakubsikora — APPROVED** [review]
1 suggestion

File: src/utils/index.js (hunk 1 above)
Comment (arturfracala) [reply src/utils/index.js:87]: It causes some webpack issues. Support for the experimental syntax 'optionalChaining' isn't currently enabled there

**Pull request merged**